README.md

```diff
@@ -127,10 +131,31 @@ Please install the ```tree``` command before moving forward:
         ```
 
    - MacOSX
-        ```
-        install tree - http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
-        install wget - http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
-        ```
+       
+        - Option 1:
+        
+          - install tree
+          
+          http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
+        
+          - install wget 
+          
+          http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
+       
+        - Option 2 (Via brew ):
+          
+            You will need the latest version of brew. If you don't have brew installed, execute the next instruction or go [here](https://brew.sh/) for more information. 
+        
+            - ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
is it possible to put this between single back ticks:
`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`